FRAuth/FRAuth/WebAuthn/Authenticator/PlatformAuthenticator/PlatformAuthenticatorGetAssertionSession.swift

```diff
@@ -294,45 +300,30 @@ class PlatformAuthenticatorGetAssertionSession: AuthenticatorGetAssertionSession
                 let context = LAContext()
                 var evalError: NSError?
                 if context.canEvaluatePolicy(.deviceOwnerAuthentication, error: &evalError) {
+                    if let error = evalError {
+                        completion(self.wakErrorForEvalError(evalError: error))
+                        return
+                    }
                     FRLog.v("Evaluation with LocalContext is available, performing biometric LocalAuthentication", subModule: WebAuthn.module)
-                    context.evaluatePolicy(.deviceOwnerAuthentication, localizedReason: NSLocalizedString(WebAuthn.localAuthenticationString, comment: "Description text for local authentication reason displayed in iOS' local authentication screen.")) { (result, error) in
-                        
+                    context.evaluatePolicy(.deviceOwnerAuthentication, localizedReason: NSLocalizedString(WebAuthn.localAuthenticationString, comment: "Description text for local authentication reason displayed in iOS' local authentication screen.")) { [weak self] (result, error) in
                         if result {
```

Comment:
can we modify this check to

if result && error != nil

Reply:
done